src/pages/contenu-libre/index.tsx

```diff
@@ -16,7 +17,7 @@ const ContenuLibre = () => {
   const { user } = React.useContext(UserContext);
   const { selectedPhase } = React.useContext(VillageContext);
 
-  const isModerator = user !== null && user.type <= UserType.MEDIATOR;
+  const isModerator = user !== null && user.type <= UserType.TEACHER;
```

Comment:
Here moderator has the same logic as Teacher.
For me, Mederator can be teacher but it could also be a larger set of admin...